includes/class-taxjar-tax-calculation.php

```diff
@@ -204,7 +205,7 @@ public function should_calculate_cart_tax( $wc_cart_object ) {
 		$should_calculate = true;
 
 		// If outside of cart and checkout page or within mini-cart, skip calculations
```

Comment:
Should this comment be removed since `should_calculate_on_page`'s body is self-documenting?

Reply:
Good catch, I've removed it.